README.md

```diff
@@ -25,3 +25,6 @@ yarn test:dev
 
 One technique that can helpful when writing tests, is to reference the coverage report for the file/function/feature you're testing. To do this, run `yarn test:dev` from your terminal. This will generate a coverage report for the project. The `coverage` directory will be located in the root directory. There is an `index.html` which shows the coverage report per file.
```

Comment:
```suggestion
One technique that can helpful when writing tests, is to reference the coverage report for the file/function/feature you're testing. To do this, run `yarn test:dev` from your terminal. This will generate a coverage report for the project in the `coverage` directory. To view the report, open the file `coverage/lcov-report/index.html`. 
```